pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -306,6 +306,10 @@ self: super: {
     '';
   };
 
+  go-nvim = super.go-nvim.overrideAttrs (old: {
+    # Makefile that runs tests I wasn't able to get working.
+    prePatch = "rm Makefile";
```

Comment:
I could try harder here, would at least need to bring in a dependency for the tests (plenary) and fixup the path to neovim so it could run.